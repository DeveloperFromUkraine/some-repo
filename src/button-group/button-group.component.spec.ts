import { ComponentFixture, TestBed } from '@angular/core/testing';
import { ButtonGroupComponent } from './index';
import { ComponentTest } from '../../test/test-bed/component';
import { Component } from '@angular/core';

describe('Button Group', () => {
  let fixture: ComponentFixture<ButtonGroupComponent>;

  beforeEach(() => {
    ComponentTest.createTestBed([], [ButtonGroupComponent] as Component[]);

    fixture = TestBed.createComponent(ButtonGroupComponent);
  });

  it('should match snapshot', () => {
    expect(fixture).toMatchSnapshot();
  });
});
