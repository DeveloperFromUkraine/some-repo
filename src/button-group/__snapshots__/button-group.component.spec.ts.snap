// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonGroupComponent should match snapshot 1`] = `
<ign-button-group>
  
</ign-button-group>
`;
